### DISCUSSION TITLE
Provide Authentication options when retrieving asset files

### DISCUSSION THREAD

**Discussion by ben-at-sahmri** [engram-design/FeedMe#1028] [Ideas]
### Description

When working with a feed, we can specify Auth details for the API endpoint itself, but (as far as I can tell), Feed Me assumes that the assets themselves are publically available.

It'd be great to be able to supply authentication options to use when retrieving assets as well.

Please let me know if I've missed something and this is available after all! Thanks!

**Comment by stigzelm** [comment]
Yes! Today I also noticed that I would need this feature. When downloading assets, preferably the fetch would use the same Auth details as the endpoint.

**Comment by rungta** [comment]
I believe this has been addressed in #1549 via the combination of `assetDownloadGuzzle` and specifying HTTP auth parameters via `clientOptions`.
